src/Common/CoverageCallbacks.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <numeric>
+#include "Coverage.h"
+
+//NOLINTNEXTLINE(bugprone-reserved-identifier, readability-non-const-parameter)
+extern "C" void __sanitizer_cov_trace_pc_guard_init(uint32_t *start, uint32_t *stop)
+{
+    if (start == stop || *start) return;
+    std::iota(start, stop, 1);
```

Comment:
Why we fill it with 1 ?

Reply:
See sanitizer documentation https://clang.llvm.org/docs/SanitizerCoverage.html#id2